docs/tutorial.rst

```diff
@@ -143,15 +141,16 @@ operations.
 How to validate schema and data migration?
 ------------------------------------------
 
-The command :ref:`pgcopydb_compare_schema` is limited to comparing the
-metadata that pgcopydb grabs about the Postgres schema at the moment. This
-means comparing the list of tables, their attributes, their indexes and
+The command :ref:`pgcopydb_compare_schema` is currently limited to comparing the
+metadata that pgcopydb grabs about the Postgres schema. This
+applies to comparing the list of tables, their attributes, their indexes and
 constraints, and the sequences values.
 
-The command :ref:`pgcopydb_compare_data` run a SQL query that computes a
-checksum of the data on each Postgres instance, for each table, and then
-only compares there checksums. This is not a full comparison of the data
-set, cases where the checksum are the same and the data differ can be found.
+The command :ref:`pgcopydb_compare_data` runs an SQL query that computes a
+checksum of the data on each Postgres instance (i.e. source and destination) 
```

Comment:
Consider using “source and target” here, as used elsewhere in the docs (see `--source` and `--target`). I believe that would be more useful than using the “destination” synonym to “target”.

Reply:
Agreed, thanks for pointing it out for consistency.